gs_quant/notebooks/Getting Started With FRED.ipynb

```diff
@@ -0,0 +1,2345 @@
+       "1947-04-01      245.968\n",
+       "1947-07-01      249.585\n",
+       "1947-10-01      259.745\n",
+       "1948-01-01      265.742\n",
+       "1948-04-01      272.567\n",
+       "1948-07-01      279.196\n",
+       "1948-10-01      280.366\n",
+       "1949-01-01      275.034\n",
+       "1949-04-01      271.351\n",
+       "1949-07-01      272.889\n",
+       "1949-10-01      270.627\n",
+       "1950-01-01      280.828\n",
+       "1950-04-01      290.383\n",
+       "1950-07-01      308.153\n",
+       "1950-10-01      319.945\n",
+       "1951-01-01        336.0\n",
+       "1951-04-01       344.09\n",
+       "1951-07-01      351.385\n",
+       "1951-10-01      356.178\n",
+       "1952-01-01       359.82\n",
+       "1952-04-01       361.03\n",
+       "1952-07-01      367.701\n",
+       "1952-10-01      380.812\n",
+       "1953-01-01       387.98\n",
+       "1953-04-01      391.749\n",
+       "1953-07-01      391.171\n",
+       "1953-10-01       385.97\n",
+       "1954-01-01      385.345\n",
+       "1954-04-01      386.121\n",
+       "1954-07-01      390.996\n",
+       "1954-10-01      399.734\n",
+       "1955-01-01      413.073\n",
+       "1955-04-01      421.532\n",
+       "1955-07-01      430.221\n",
+       "1955-10-01      437.092\n",
+       "1956-01-01      439.746\n",
+       "1956-04-01       446.01\n",
+       "1956-07-01      451.191\n",
+       "1956-10-01      460.463\n",
+       "1957-01-01      469.779\n",
+       "1957-04-01      472.025\n",
+       "1957-07-01       479.49\n",
+       "1957-10-01      474.864\n",
+       "1958-01-01       467.54\n",
+       "1958-04-01      471.978\n",
+       "1958-07-01      485.841\n",
+       "1958-10-01      499.555\n",
+       "1959-01-01       510.33\n",
+       "1959-04-01      522.653\n",
+       "1959-07-01      525.034\n",
+       "1959-10-01        528.6\n",
+       "1960-01-01      542.648\n",
+       "1960-04-01       541.08\n",
+       "1960-07-01      545.604\n",
+       "1960-10-01      540.197\n",
+       "1961-01-01      545.018\n",
+       "1961-04-01      555.545\n",
+       "1961-07-01      567.664\n",
+       "1961-10-01      580.612\n",
+       "1962-01-01      594.013\n",
+       "1962-04-01      600.366\n",
+       "1962-07-01      609.027\n",
+       "1962-10-01       612.28\n",
+       "1963-01-01      621.672\n",
+       "1963-04-01      629.752\n",
+       "1963-07-01      644.444\n",
+       "1963-10-01      653.938\n",
+       "1964-01-01      669.822\n",
+       "1964-04-01      678.674\n",
+       "1964-07-01      692.031\n",
+       "1964-10-01      697.319\n",
+       "1965-01-01       717.79\n",
+       "1965-04-01      730.191\n",
+       "1965-07-01      749.323\n",
+       "1965-10-01      771.857\n",
+       "1966-01-01      795.734\n",
+       "1966-04-01      804.981\n",
+       "1966-07-01      819.638\n",
+       "1966-10-01      833.302\n",
+       "1967-01-01       844.17\n",
+       "1967-04-01      848.983\n",
```

Comment:
let's remove the notebook for now - after released we can work on coming up with a few bite sized examples